.travis.yml

```diff
@@ -27,6 +27,7 @@ script:
   - ineffassign .
   - go vet ./...
   - go test -race ./...
+  - $HOME/gopath/bin/golint ./...
 
 after_script:
   - golint ./...
```

Comment:
Why do we run golint during and after? I also don't think you'll need the full path like that.